syft/formats/common/spdxhelpers/source_info.go

```diff
@@ -54,6 +54,8 @@ func SourceInfo(p pkg.Package) string {
 		answer = "acquired package info from nix store path"
 	case pkg.Rpkg:
 		answer = "acquired package info from R-package DESCRIPTION file"
+	case pkg.SwiftPkg:
+		answer = "acquired package info from resolved Swift package manifest"
```

Comment:
Is this description correct?

I'm pretty inexperienced with the Swift ecosystem, but the docs I've read describe the package manifest file as the `Package.swift` file. When I run the branch against a swift project I see  packages sourced from `Package.resolved` <-- is there a different name for this file or is this also a manifest?

Reply:
The `Package.swift` file describes a package (i.e something that would be distributed with Swift Package Manager), meanwhile the `Package.resolved` file describes dependencies that the project depends on. The `Package.resolved` file is the result of looking up the dependencies of the given project, so it's a standalone manifest.